<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1136</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1131</width>
      <height>581</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>301</width>
        <height>411</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox{

color:rgb(24, 50, 109);
	font: 10pt &quot;MS Shell Dlg 2&quot;;
  



    border: 3px solid white;



}</string>
      </property>
      <property name="title">
       <string>Formulaire</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
font: 63 10pt &quot;Segoe UI Semibold&quot;;
color : rgb(24, 50, 109);
type :italic;
}</string>
       </property>
       <property name="text">
        <string>Numero:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
font: 63 10pt &quot;Segoe UI Semibold&quot;;
color : rgb(24, 50, 109);
}</string>
       </property>
       <property name="text">
        <string>Nom:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
font: 63 10pt &quot;Segoe UI Semibold&quot;;
color : rgb(24, 50, 109);
}</string>
       </property>
       <property name="text">
        <string>Type:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
font: 63 10pt &quot;Segoe UI Semibold&quot;;
color : rgb(24, 50, 109);
}</string>
       </property>
       <property name="text">
        <string>Date de Debut:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
font: 63 10pt &quot;Segoe UI Semibold&quot;;
color : rgb(24, 50, 109);
}</string>
       </property>
       <property name="text">
        <string>Date de Fin:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>250</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
font: 63 10pt &quot;Segoe UI Semibold&quot;;
color : rgb(24, 50, 109);
}</string>
       </property>
       <property name="text">
        <string>Prix:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ajouter">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>320</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
border-radius:15px;
font: 10pt &quot;MS Shell Dlg 2&quot;;
border-right :1px solid #aaaaaa;
border-bottom :1px solid #aaaaaa;
}
QPushButton:enabled {
background-color: #E1F4FF;
color: #0c2f70;
}
QPushButton:pressed {
background-color:blue;
color: white;
}

QPushButton:hover:!pressed {

background-color : #0d2f72;
color: #fffffe;

}
QPushButton:disabled{
boackground-color:#aaaaaa;
color :ffffff;
}</string>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="numero">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>50</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="nom">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>90</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="prix">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>250</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="modifier">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>320</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
border-radius:15px;
font: 10pt &quot;MS Shell Dlg 2&quot;;
border-right :1px solid #aaaaaa;
border-bottom :1px solid #aaaaaa;
}
QPushButton:enabled {
background-color: #E1F4FF;
color: #0c2f70;
}
QPushButton:pressed {
background-color:blue;
color: white;
}

QPushButton:hover:!pressed {

background-color : #0d2f72;
color: #fffffe;

}
QPushButton:disabled{
boackground-color:#aaaaaa;
color :ffffff;
}</string>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="date_debut">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>170</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QDateEdit" name="date_fin">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>210</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="type">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>131</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>cosmetique</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>alimentaire</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>industriel</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QLineEdit" name="chercher_num">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>130</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>40</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	font: 63 10pt &quot;Segoe UI Semibold&quot;;
 

color:rgb(24, 50, 109);
}
</string>
      </property>
      <property name="text">
       <string>Tri par :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>100</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	font: 63 10pt &quot;Segoe UI Semibold&quot;;
 

color:rgb(24, 50, 109);
}
</string>
      </property>
      <property name="text">
       <string>Recherche par numero :</string>
      </property>
     </widget>
     <widget class="QTableView" name="tab_recherche">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>200</y>
        <width>701</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="supp">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>450</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_supp">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>450</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
border-radius:15px;
font: 10pt &quot;MS Shell Dlg 2&quot;;
border-right :1px solid #aaaaaa;
border-bottom :1px solid #aaaaaa;
}
QPushButton:enabled {
background-color: #E1F4FF;
color: #0c2f70;
}
QPushButton:pressed {
background-color:blue;
color: white;
}

QPushButton:hover:!pressed {

background-color : #0d2f72;
color: #fffffe;

}
QPushButton:disabled{
boackground-color:#aaaaaa;
color :ffffff;
}</string>
      </property>
      <property name="text">
       <string>supp</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>460</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
font: 63 10pt &quot;Segoe UI Semibold&quot;;
color : rgb(24, 50, 109);
}</string>
      </property>
      <property name="text">
       <string>Numero a supprimé</string>
      </property>
     </widget>
     <widget class="QPushButton" name="tri_num">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>30</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
border-radius:15px;
font: 10pt &quot;MS Shell Dlg 2&quot;;
border-right :1px solid #aaaaaa;
border-bottom :1px solid #aaaaaa;
}
QPushButton:enabled {
background-color: #E1F4FF;
color: #0c2f70;
}
QPushButton:pressed {
background-color:blue;
color: white;
}

QPushButton:hover:!pressed {

background-color : #0d2f72;
color: #fffffe;

}
QPushButton:disabled{
boackground-color:#aaaaaa;
color :ffffff;
}</string>
      </property>
      <property name="text">
       <string>Numero</string>
      </property>
     </widget>
     <widget class="QPushButton" name="tri_prix">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>80</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
border-radius:15px;
font: 10pt &quot;MS Shell Dlg 2&quot;;
border-right :1px solid #aaaaaa;
border-bottom :1px solid #aaaaaa;
}
QPushButton:enabled {
background-color: #E1F4FF;
color: #0c2f70;
}
QPushButton:pressed {
background-color:blue;
color: white;
}

QPushButton:hover:!pressed {

background-color : #0d2f72;
color: #fffffe;

}
QPushButton:disabled{
boackground-color:#aaaaaa;
color :ffffff;
}</string>
      </property>
      <property name="text">
       <string>Prix</string>
      </property>
     </widget>
     <widget class="QPushButton" name="tri_date">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>130</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
border-radius:15px;
font: 10pt &quot;MS Shell Dlg 2&quot;;
border-right :1px solid #aaaaaa;
border-bottom :1px solid #aaaaaa;
}
QPushButton:enabled {
background-color: #E1F4FF;
color: #0c2f70;
}
QPushButton:pressed {
background-color:blue;
color: white;
}

QPushButton:hover:!pressed {

background-color : #0d2f72;
color: #fffffe;

}
QPushButton:disabled{
boackground-color:#aaaaaa;
color :ffffff;
}</string>
      </property>
      <property name="text">
       <string>Date de Fin</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>100</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	font: 63 10pt &quot;Segoe UI Semibold&quot;;
 

color:rgb(24, 50, 109);
}
</string>
      </property>
      <property name="text">
       <string>Recherche par prix :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="chercher_prix">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>130</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <zorder>groupBox</zorder>
     <zorder>label_7</zorder>
     <zorder>chercher_num</zorder>
     <zorder>label_8</zorder>
     <zorder>tab_recherche</zorder>
     <zorder>supp</zorder>
     <zorder>pb_supp</zorder>
     <zorder>label_9</zorder>
     <zorder>tri_num</zorder>
     <zorder>tri_prix</zorder>
     <zorder>tri_date</zorder>
     <zorder>label_10</zorder>
     <zorder>chercher_prix</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QPushButton" name="retour">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>410</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
border-radius:15px;
font: 10pt &quot;MS Shell Dlg 2&quot;;
border-right :1px solid #aaaaaa;
border-bottom :1px solid #aaaaaa;
}
QPushButton:enabled {
background-color: #E1F4FF;
color: #0c2f70;
}
QPushButton:pressed {
background-color:blue;
color: white;
}

QPushButton:hover:!pressed {

background-color : #0d2f72;
color: #fffffe;

}
QPushButton:disabled{
boackground-color:#aaaaaa;
color :ffffff;
}</string>
      </property>
      <property name="text">
       <string>Quitter</string>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>440</y>
        <width>621</width>
        <height>31</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="start">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>380</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
border-radius:15px;
font: 10pt &quot;MS Shell Dlg 2&quot;;
border-right :1px solid #aaaaaa;
border-bottom :1px solid #aaaaaa;
}
QPushButton:enabled {
background-color: #E1F4FF;
color: #0c2f70;
}
QPushButton:pressed {
background-color:blue;
color: white;
}

QPushButton:hover:!pressed {

background-color : #0d2f72;
color: #fffffe;

}
QPushButton:disabled{
boackground-color:#aaaaaa;
color :ffffff;
}</string>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pause">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>380</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
border-radius:15px;
font: 10pt &quot;MS Shell Dlg 2&quot;;
border-right :1px solid #aaaaaa;
border-bottom :1px solid #aaaaaa;
}
QPushButton:enabled {
background-color: #E1F4FF;
color: #0c2f70;
}
QPushButton:pressed {
background-color:blue;
color: white;
}

QPushButton:hover:!pressed {

background-color : #0d2f72;
color: #fffffe;

}
QPushButton:disabled{
boackground-color:#aaaaaa;
color :ffffff;
}</string>
      </property>
      <property name="text">
       <string>Pause</string>
      </property>
     </widget>
     <widget class="QPushButton" name="stat">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>50</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
border-radius:15px;
font: 10pt &quot;MS Shell Dlg 2&quot;;
border-right :1px solid #aaaaaa;
border-bottom :1px solid #aaaaaa;
}
QPushButton:enabled {
background-color: #E1F4FF;
color: #0c2f70;
}
QPushButton:pressed {
background-color:blue;
color: white;
}

QPushButton:hover:!pressed {

background-color : #0d2f72;
color: #fffffe;

}
QPushButton:disabled{
boackground-color:#aaaaaa;
color :ffffff;
}</string>
      </property>
      <property name="text">
       <string>Statistique</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Facture">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>110</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
border-radius:15px;
font: 10pt &quot;MS Shell Dlg 2&quot;;
border-right :1px solid #aaaaaa;
border-bottom :1px solid #aaaaaa;
}
QPushButton:enabled {
background-color: #E1F4FF;
color: #0c2f70;
}
QPushButton:pressed {
background-color:blue;
color: white;
}

QPushButton:hover:!pressed {

background-color : #0d2f72;
color: #fffffe;

}
QPushButton:disabled{
boackground-color:#aaaaaa;
color :ffffff;
}</string>
      </property>
      <property name="text">
       <string>Facture</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pdf">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>170</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
border-radius:15px;
font: 10pt &quot;MS Shell Dlg 2&quot;;
border-right :1px solid #aaaaaa;
border-bottom :1px solid #aaaaaa;
}
QPushButton:enabled {
background-color: #E1F4FF;
color: #0c2f70;
}
QPushButton:pressed {
background-color:blue;
color: white;
}

QPushButton:hover:!pressed {

background-color : #0d2f72;
color: #fffffe;

}
QPushButton:disabled{
boackground-color:#aaaaaa;
color :ffffff;
}</string>
      </property>
      <property name="text">
       <string>PDF</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_excel">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>230</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
border-radius:15px;
font: 10pt &quot;MS Shell Dlg 2&quot;;
border-right :1px solid #aaaaaa;
border-bottom :1px solid #aaaaaa;
}
QPushButton:enabled {
background-color: #E1F4FF;
color: #0c2f70;
}
QPushButton:pressed {
background-color:blue;
color: white;
}

QPushButton:hover:!pressed {

background-color : #0d2f72;
color: #fffffe;

}
QPushButton:disabled{
boackground-color:#aaaaaa;
color :ffffff;
}</string>
      </property>
      <property name="text">
       <string>Excel</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber_2">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>0</y>
        <width>151</width>
        <height>91</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber_1">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>0</y>
        <width>151</width>
        <height>91</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="camera">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>290</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
border-radius:15px;
font: 10pt &quot;MS Shell Dlg 2&quot;;
border-right :1px solid #aaaaaa;
border-bottom :1px solid #aaaaaa;
}
QPushButton:enabled {
background-color: #E1F4FF;
color: #0c2f70;
}
QPushButton:pressed {
background-color:blue;
color: white;
}

QPushButton:hover:!pressed {

background-color : #0d2f72;
color: #fffffe;

}
QPushButton:disabled{
boackground-color:#aaaaaa;
color :ffffff;
}</string>
      </property>
      <property name="text">
       <string>camera</string>
      </property>
     </widget>
     <widget class="QPushButton" name="capture2">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>100</y>
        <width>311</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
border-radius:15px;
font: 10pt &quot;MS Shell Dlg 2&quot;;
border-right :1px solid #aaaaaa;
border-bottom :1px solid #aaaaaa;
}
QPushButton:enabled {
background-color: #E1F4FF;
color: #0c2f70;
}
QPushButton:pressed {
background-color:blue;
color: white;
}

QPushButton:hover:!pressed {

background-color : #0d2f72;
color: #fffffe;

}
QPushButton:disabled{
boackground-color:#aaaaaa;
color :ffffff;
}</string>
      </property>
      <property name="text">
       <string>Enregistrer temps</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>380</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	font: 63 10pt &quot;Segoe UI Semibold&quot;;
 

color:rgb(24, 50, 109);
}
</string>
      </property>
      <property name="text">
       <string>Musique:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>430</y>
        <width>81</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	font: 63 10pt &quot;Segoe UI Semibold&quot;;
 

color:rgb(24, 50, 109);
}
</string>
      </property>
      <property name="text">
       <string>Volume:</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1136</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
